### ISSUE TITLE
Support Translations

### ISSUE THREAD

**Issue by Prayag2** [Prayag2/drawy#63]
**Is your feature request related to a problem? Please describe.**  
Currently, our platform lacks translation support, which limits accessibility for non-native speakers and reduces the potential user base.
